neo3/wallet/scrypt_parameters.py

```diff
@@ -0,0 +1,15 @@
+class ScryptParameters(object):
+
+    instance = None
+
+    @staticmethod
+    def default():
+        if ScryptParameters.instance is None:
+            ScryptParameters.instance = ScryptParameters(16384, 8, 8, 64)
+        return ScryptParameters.instance
+
+    def __init__(self, n, r, p, length):
```

Comment:
1. add types
2. idea; instead of having a `default`, have the initialiser arguments be optional and default them to 16384, 8, 8 , 64